components/prism-typoscript.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.typoscript = Prism.languages.extend('javascript', {
+    'comment': /(?:\s|^)[#][^#^\r\n]{2,}?(?:\r?\n|$)|(?:\s|^)\/\/.*(?:\r?\n|$)/g,
```

Comment:
The preceding space shouldn't be part of the token.

Also, you don't need a `g` flag in any patterns. If Prism needs it, it will be added.

```suggestion
	'comment': {
		pattern: /(^|\s)(?:#[^#^\r\n]{2,}(?:\r?\n|$)|\/\/.*)/,
		lookbehind: true
	},
```

Reply:
I've done a complete rewrite here, i will add this asap.